messaginginapp/src/main/java/io/customer/messaginginapp/gist/data/listeners/Queue.kt

```diff
@@ -69,6 +69,8 @@ class Queue : GistQueue {
                 val networkRequest = originalRequest.newBuilder()
                     .addHeader(NetworkUtilities.CIO_SITE_ID_HEADER, state.siteId)
                     .addHeader(NetworkUtilities.CIO_DATACENTER_HEADER, state.dataCenter)
+                    .addHeader(NetworkUtilities.CIO_CLIENT_PLATFORM, SDKComponent.android().client.source.lowercase())
+                    .addHeader(NetworkUtilities.CIO_CLIENT_VERSION, SDKComponent.android().client.sdkVersion)
```

Comment:
Did you verify if React Native Data Pipelines support is sending expected source and version correctly? If yes, then we're good with the change.